Sources/ATTNPersistentStorage.m

```diff
@@ -0,0 +1,40 @@
+//
+//  ATTNPersistentStorage.m
+//  Example
+//
+//  Created by Olivia Kim on 11/23/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "ATTNPersistentStorage.h"
+
+
+NSString * const ATTN_PERSISTENT_STORAGE_PREFIX = @"com.attentive.iossdk.PERSISTENT_STORAGE";
+
+
+@implementation ATTNPersistentStorage
+
+
+- (id)init {
+    _userDefaults = [NSUserDefaults standardUserDefaults];
```

Comment:
I know this is super weird, but the general pattern for an Objective-C `init` method is:

```
- (id)init {
  if (self = [super init]) {
    // any setup (like setting instance variables)
  }
  return self
}
```

It's possible the `[super init]` could return a different object instance, which is why it should be called first. 

Docs [here](https://developer.apple.com/documentation/objectivec/nsobject/1418641-init) on it.